lib/DDG/Spice/GetEvents/Nearby.pm

```diff
@@ -1,15 +1,15 @@
-package DDG::Spice::GetEvents;
+package DDG::Spice::GetEvents::Nearby;
```

Comment:
@moollaza Since the different queries have both different caching requirement and api structures it made more sense for me to implement the new functionality in a different spice (file).The code is cleaner this way on the pearl side but duplicated on the front end. 
If I change it would the program be able to handle both caching requirements on the same file?